src/mame/namco/namconb1.h

```diff
@@ -39,44 +38,31 @@ class namconb1_state : public driver_device
 		m_p3(*this, "P3"),
 		m_p4(*this, "P4"),
 		m_misc(*this, "MISC"),
-		m_light0_x(*this, "LIGHT0_X"),
-		m_light0_y(*this, "LIGHT0_Y"),
-		m_light1_x(*this, "LIGHT1_X"),
-		m_light1_y(*this, "LIGHT1_Y"),
 		m_spritebank32(*this, "spritebank32"),
-		m_tilebank32(*this, "tilebank32"),
-		m_rozbank32(*this, "rozbank32"),
 		m_namconb_shareram(*this, "namconb_share"),
 		m_update_to_line_before_posirq(false)
 	{ }
 
 	void namconb1(machine_config &config);
-	void namconb2(machine_config &config);
-	void outfxies(machine_config &config);
-	void machbrkr(machine_config &config);
 
 	void init_sws95();
-	void init_machbrkr();
 	void init_sws97();
 	void init_sws96();
 	void init_vshoot();
 	void init_nebulray();
-	void init_gunbulet();
 	void init_gslgr94j();
-	void init_outfxies();
 	void init_gslgr94u();
 
 protected:
 	virtual void machine_start() override ATTR_COLD;
 	virtual void machine_reset() override ATTR_COLD;
 	virtual void video_start() override ATTR_COLD;
 
-private:
 	int m_gametype = 0;
 	enum
 	{
 		/* Namco NB1 */
-		NAMCONB1_NEBULRAY = 0x2000,
+		NAMCONB1_NEBULRAY = 0,
```

Comment:
This changes behaviour so it defaults to applying hacks for Nebulas Ray if the game type is not set.  Previously, no hacks would be applied if the game type was not set (keeping initial zero value).  The default value zero should not correspond to any per-game hack configuration.